script/1_0_2current_en_script/Arcueid/Day 5/05_00_ARC05_6_2.txt

```diff
@@ -5,58 +5,58 @@ The ground below is completely filled with blue flames.
 A tall vampire stands at the center.
 Vlov Arkhangel.
 He stands in the middle of the crater without a single scratch.
-By human standards, he's basically an immortal monster.
+By human standards, he's practically an immortal monster.
 And then...
 <Page3>
-There is a white figure trifling with the man from the front.
-The beautiful hunter runs through the garden of flickering flames and debris.
+There is a white figure toying with the man from the front.
+The beautiful hunter darts through the garden of flickering flames and debris.
 <Page4>
 Of course, that white figure belongs to Arcueid.
 She leaps across the river of flames and closes in on Vlov.
 <Page5>
-A crude and deadly weapon is gripped in Vlov's hand.
+He has a crude and deadly weapon gripped in his hand.
 Although it has the appearance of a long sword, the blade is that of a billhook.
 Arcueid on the other hand is fighting empty-handed.
 Using the force of her legs alone, she closes in on Vlov. And then...
 <Page6>
 She exchanges blows with Vlov's billhook using just her claws, causing the tall figure to retreat...!
 <Page7>
-A single wound can be seen on Vlov's arm, it is so deep that the bone underneath is visible.
-It was only visible for an instant. The arm that had been sliced open by Arcueid's claw heals thanks to the blood pouring out.
+A single wound can be seen on his arm, it's so deep that the bone within is visible.
+It was only visible for an instant. The arm that had been sliced open by Arcueid's claw is healed by outpour of blood.
```

Comment:
A single wound can be seen on his arm, so deep that the bone within is visible.
It is only there for an instant. The arm sliced open by Arcueid's claw is healed by the very blood that pours forth.